persist/src/main/resources/db/migration/V1__Initial_Import.sql

```diff
@@ -9,9 +18,99 @@ CREATE TABLE IF NOT EXISTS users
     first_name      VARCHAR(36) NOT NULL,
     last_name       VARCHAR(36) NOT NULL,
     privilege_level VARCHAR(16) NOT NULL,
-    password_hash   BYTEA       NOT NULL
+    country_id      BIGINT,
+    password_hash   BYTEA       NOT NULL,
+
+    CONSTRAINT fk_user_country FOREIGN KEY (country_id) REFERENCES countries (id)
+);
+
+CREATE TABLE IF NOT EXISTS schools
+(
+    id         BIGSERIAL   NOT NULL PRIMARY KEY,
+    created_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    updated_at TIMESTAMP   NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    deleted_at TIMESTAMP,
+    name       VARCHAR(36) NOT NULL,
+    address    VARCHAR(64) NOT NULL,
+    hidden     BOOLEAN              DEFAULT FALSE,
+    country_id BIGINT,
```

Comment:
Isn't `countries.country_id` a serial?

Reply:
Also, should this column be nullable?

Reply:
I think a school should **always** be tied to a country--why would we want it `nullable`?